software/contrib/noddy_holder.py

```diff
@@ -0,0 +1,138 @@
+from europi import *
+import machine
+from europi_script import EuroPiScript
+from time import ticks_diff, ticks_ms
+
+'''
+NoddyHolder
+author: Sean Bechhofer (github.com/seanbechhofer)
+date: 2022-03-26
+labels: sample & hold, trigger & hold
+
+Two channels of sample and hold/trigger and hold. 
+
+Digital in: Gate
+Analog in: CV
+
+Output 1: Gate
+Output 2: Sample and Hold based on Gate and CV
+Output 3: Trigger and Hold based on Gate and CV
+
+Output 4: Inverted Gate
+Output 5: Sample and Hold based on Inverted Gate and CV
+Output 6: Trigger and Hold based on Inverted Gate and CV
+
+digital_in: gate
+analog_in: cv
+
+knob_1: Not used
+knob_2: Not used
+
+button_1: Not used
+button_2: Not used
+
+output_1: gate 
+output_2: s&h
+output_3: t&h
+output_4: inverted gate
+output_5: s&h using inverted gate
+output_6: t&h using inverted gate
+
+'''
+# Version number
+
+VERSION="1.0"
+
+# Overclock the Pico for improved performance.
+machine.freq(250_000_000)
+
+class NoddyHolder(EuroPiScript):
```

Comment:
I see you've implemented `EuroPiScript` here, is it your intention that this script be added to the menu? If so you'll just need to add it to `menu.py` as described in `menu.md`. If not it's fine as is.